.github/scripts/lernaDeps.json

```diff
@@ -0,0 +1,32 @@
+{
+    "common": [
+        "packages/eslint-config"
+    ],
+    "cli": [
+        "packages/manifest",
+        "packages/api",
+        "packages/fx-core",
+        "packages/cli",
+        "templates"
+    ],
+    "vscode": [
+        "packages/manifest",
+        "packages/api",
+        "packages/fx-core",
+        "packages/vscode-extension",
+        "templates"
+    ],
+    "server": [
+        "packages/manifest",
+        "packages/api",
+        "packages/fx-core",
+        "templates"
+    ],
+    "sdk": [
+        "packages/sdk",
```

Comment:
if sdk is the input of pkg option, what's the expected behavior of sdk version in template?

Reply:
template will not sync up version with sdk
if you want to update the sdk version in template, you may input " sdk template"

Reply:
I see, thank you!